.github/actions/setup-env/action.yml

```diff
@@ -1,54 +1,53 @@
 name: "Set up environment"
 description: ""
-inputs: {}
-outputs: {}
 runs:
   using: composite
   steps:
-    - uses: cachix/install-nix-action@v14
+    - name: Install nix
+      uses: nixbuild/nix-quick-install-action@v19
       with:
-        install_url: https://nixos-nix-install-tests.cachix.org/serve/vij683ly7sl95nnhb67bdjjfabclr85m/install
-        install_options: "--tarball-url-prefix https://nixos-nix-install-tests.cachix.org/serve"
-        extra_nix_config: |
-          experimental-features = nix-command flakes
+        nix_conf: experimental-features = nix-command flakes
 
-    - run: nix flake check
+    - name: Add nix devshell to .bash_profile
+      run: echo "source <(nix print-dev-env --show-trace)" >> "$HOME/.bash_profile"
```

Comment:
Stole this from Gadget's CI. This lets us use nix stuff without `nix develop -c`